2016/07/08 13:37:30 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:37:32 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:37:32 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:37:33 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:37:33 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:38:13 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:38:13 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:38:27 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET /web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:39:19 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET /web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:39:20 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET /web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:39:20 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET /web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:39:23 [error] 24872#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:42:14 [error] 24872#0: *15 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:42:15 [error] 24872#0: *15 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:44:22 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:44:22 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:44:23 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:44:23 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:44:23 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:01 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:01 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:02 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:02 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:02 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:17 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:18 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:18 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:18 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:45:19 [error] 24872#0: *18 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:47:08 [error] 24872#0: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:47:09 [error] 24872#0: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2016/07/08 13:47:09 [error] 24872#0: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
